src/core/1/plugins/guides/events/generic-events/index.md

```diff
@@ -0,0 +1,305 @@
+---
+type: page
+code: false
+title: Generic Events
+order: 150
+---
+
+# Generic Events
+
+Some actions in the document controller triggers generic events. Generic events are used to apply modifications on the documents in the request or result of these actions.
```

Comment:
```suggestion
Some actions in the document controller trigger generic events. Generic events are used to apply modifications on the documents in the request or result of these actions.
```

Reply:
> Generic events are used to apply modifications on the documents in the request or result of these actions.

Could you explain the meaning of this?

Reply:
I explain that the utility of generic events is to modify the documents (containing in the request or the result) like a middleware